konan/konan.properties

```diff
@@ -200,6 +200,44 @@ abiSpecificLibraries.linux_arm32_hfp = \
     lib/arm-linux-gnueabihf \
     usr/lib/arm-linux-gnueabihf
 
+# FIRST Robotics Competition
+targetToolchain-macos_x64-frc_arm32.default = \
+  /usr/local/arm-frc-linux-gnueabi
+libffi-3.2.1-frc_arm32.default = \
+  /Users/kunalsheth/Documents/konanRIO/libffi-3.2.1-frc_arm32
+target-sysroot-frc_arm32.default = \
+  /usr/local/arm-frc-linux-gnueabi
+target-gcc-toolchain-macos_x64.default = \
+  /usr/bin/gcc
```

Comment:
I am not sure why this is necessary, but it seems like [`ClangArgs.kt`](https://github.com/JetBrains/kotlin-native/blob/0da110a7869676ed0a4b18597f4c55a04d7246f0/shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/ClangArgs.kt#L162) depends on it.